.testcafe/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:9.11.2-stretch
+LABEL maintainer="dufresne.david.2@gmail.com"
+
+RUN npm install -g testcafe@0.23.2
+
+RUN echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list && \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4397703)